docs/release-notes/rockcraft-1-8-0.rst

```diff
@@ -0,0 +1,112 @@
+.. _release-1.8.0:
+
+Rockcraft 1.8.0 release notes
+=============================
+
+31 January 2025
+
+Learn about the new features, changes, and fixes introduced in Rockcraft 1.8.0.
+For information about the Rockcraft release cycle, see the
+:ref:`release_policy_and_schedule`.
+
+
+Requirements and compatibility
+------------------------------
+
+To run Rockcraft, a system requires the following minimum hardware and
+installed software. These requirements apply to local hosts as well as VMs and
+container hosts.
+
+
+Minimum hardware requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- AMD64, ARM64, ARMv7-M, RISC-V 64-bit, PowerPC 64-bit little-endian, or S390x
+  processor
+- 2GB RAM
+- 10GB available storage space
+- Internet access for remote software sources and the Snap Store
+
+
+Platform requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+  :header-rows: 1
+  :widths: 1 3 3
+
+  * - Platform
+    - Version
+    - Software requirements
+  * - GNU/Linux
+    - Popular distributions that ship with systemd and are `compatible with
+      snapd <https://snapcraft.io/docs/installing-snapd>`_
+    - systemd
+
+
+What's new
+----------
+
+Rockcraft 1.8.0 brings the following features, integrations, and improvements.
+
+Improved error reporting
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+We improved the reporting of build-time errors to emphasize the precise cause
+of the error. For example, here's the output of ``rockcraft build`` when trying
+to compile curl with incorrect build options:
+
+
+.. code-block:: shell
+
+    rockcraft build
```

Comment:
```suggestion
.. terminal::

```

Since we said that this is the output, we shouldn't include the input. Generally the better practice is to separate them. For output we have the handy `terminal` directive.